thunderbird/processes/wps_split_merged_climos.py

```diff
@@ -0,0 +1,118 @@
+# Processor imports
+from pywps import (
+    Process,
+    LiteralInput,
+    ComplexInput,
+    ComplexOutput,
+    FORMATS,
+)
+from pywps.app.Common import Metadata
+
+# Tool imports
+from nchelpers import CFDataset
+from dp.split_merged_climos import split_merged_climos
+from thunderbird.utils import (
+    MAX_OCCURS,
+    is_opendap_url,
+    collect_output_files,
+    build_meta_link,
+)
+
+# Library imports
+import logging
+import os
+import math
+
+logger = logging.getLogger("PYWPS")
+
+
+class SplitMergedClimos(Process):
+    def __init__(self):
+        self.log_level_choices = [
+            "NOTSET",
+            "DEBUG",
+            "INFO",
+            "WARNING",
+            "ERROR",
+            "CRITICAL",
+        ]
+        inputs = [
+            ComplexInput(
+                "netcdf",
+                "NetCDF Datasets",
+                abstract="NetCDF files to process",
+                min_occurs=1,
+                max_occurs=MAX_OCCURS,
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+            ),
+            LiteralInput(
+                "loglevel",
+                "Log Level",
+                default="INFO",
+                abstract="Logging level",
+                allowed_values=self.log_level_choices,
+            ),
+        ]
+        outputs = [
+            ComplexOutput(
+                "output",
+                "Output",
+                abstract="Collection of split climatological files",
+                as_reference=True,
+                supported_formats=[FORMATS.META4],
+            ),
+        ]
+
+        super(SplitMergedClimos, self).__init__(
+            self._handler,
+            identifier="split_merged_climos",
+            title="Split Merged Climos",
+            abstract="Split climo means files into one file per time interval",
+            metadata=[
+                Metadata("NetCDF processing"),
+                Metadata("Climate Data Operations"),
+            ],
+            inputs=inputs,
+            outputs=outputs,
+            store_supported=True,
+            status_supported=True,
+        )
+
+    def get_filepaths(self, request):
+        filepaths = []
+        for path in request.inputs["netcdf"]:
+            if is_opendap_url(path.url):
+                filepaths.append(path.url)
+            elif path.file.endswith(".nc"):
+                filepaths.append(path.file)
+            else:
+                raise ProcessError(
+                    "You must provide a data source (opendap/netcdf). "
+                    f"Inputs provided: {request.inputs}"
+                )
+        return filepaths
+
+    def _handler(self, request, response):
+        response.update_status("Starting Process", 0)
```

Comment:
I don't think this is the place to address the issue but I do think we need to do a better job of logging the process.  From what I understand the `response.update_status()` method is mainly used by `phoenix` for logging.  It doesn't really do much for us though since I don't think it outputs to the terminal. I might make an issue for it.